hasura/seeds/carnet_de_bord/seed-data.sql

```diff
@@ -186,7 +186,8 @@ INSERT INTO public.notebook (id, beneficiary_id, created_at, right_rsa, right_rq
 VALUES ('24e335cb-4e2b-481b-84b7-617d77f60f56', '3c1a9fdf-a231-4659-9a91-630ff12c5774', '2022-03-30 09:29:36.22194+00', NULL, false, false, true, false, NULL, NULL, NULL, NULL, NULL, NULL, '2022-03-30 09:29:36.22194+00');
 INSERT INTO public.notebook_member (id, notebook_id, account_id, last_visited_at, member_type, created_at, creator_id, invitation_sent_at) VALUES ('f96735c3-dcbd-4115-88e0-b1364ff193e3', '24e335cb-4e2b-481b-84b7-617d77f60f56', '2addd10f-9bd3-4d37-b3c9-10a6e2c4be4f', '2021-09-21 13:06:45.076+00', 'orientation_manager', '2021-09-21 11:51:37.295647+00', '9eee9fea-bf3e-4eb8-8f43-d9b7fd6fae76', NULL);
 INSERT INTO public.notebook_member (id, notebook_id, account_id, last_visited_at, member_type, created_at, creator_id, invitation_sent_at) VALUES ('87268795-0541-48d2-8d59-0267c787432a', '24e335cb-4e2b-481b-84b7-617d77f60f56', '521a2983-17e2-43a9-abf6-7ed9f93cf7f9', '2021-09-21 13:06:45.076+00', 'referent', '2021-09-21 11:51:37.295647+00', '9eee9fea-bf3e-4eb8-8f43-d9b7fd6fae76', NULL);
-INSERT INTO public.orientation_request (id, beneficiary_id, requestor_account_id, requested_orientation_type_id, status, reason, created_at) values ('eabf6fda-26b5-4aeb-8179-6050a826cde6', '3c1a9fdf-a231-4659-9a91-630ff12c5774', '521a2983-17e2-43a9-abf6-7ed9f93cf7f9', 'social', 'current', 'Ceci est la raison pour laquelle on souhaite changer l’orientation', '2022-09-01 14:00:00.0000+00');
+INSERT INTO public.orientation_request (id, beneficiary_id, requestor_account_id, requested_orientation_type_id, status, reason, created_at)
+values ('eabf6fda-26b5-4aeb-8179-6050a826cde6', '3c1a9fdf-a231-4659-9a91-630ff12c5774', '521a2983-17e2-43a9-abf6-7ed9f93cf7f9', 'social', 'pending', 'Ceci est la raison pour laquelle on souhaite changer l’orientation', '2022-09-01 14:00:00.0000+00');
```

Comment:
Suggestion : garder l'`INSERT` sur une seule ligne, et mettre `VALUES` en majuscules comme ailleurs.
```suggestion
INSERT INTO public.orientation_request (id, beneficiary_id, requestor_account_id, requested_orientation_type_id, status, reason, created_at) VALUES ('eabf6fda-26b5-4aeb-8179-6050a826cde6', '3c1a9fdf-a231-4659-9a91-630ff12c5774', '521a2983-17e2-43a9-abf6-7ed9f93cf7f9', 'social', 'pending', 'Ceci est la raison pour laquelle on souhaite changer l’orientation', '2022-09-01 14:00:00.0000+00');
```

Reply:
comme il y a deja des insert sur 2 lignes je fais ca partout

Reply:
@jonathanperret l'idée des inserts sur deux lignes, c'était de pouvoir lire les valeurs qu'on insère facilement dans le fichier, sinon on se retrouve avec une partie gauche du fichier sans aucune information utile.
Les inserts avec des informations importantes (beneficiary_id, account_id, notebook_id) sont sur deux lignes, certains autres sont sur une ligne.